lib/Definition/EntityDefinition.json

```diff
@@ -240,6 +240,21 @@
 		"model"			: "Volkszaehler\\Model\\Channel",
 		"hasConsumption"	: true,
 		"translation"		: {
+			"de" : "Betriebsimpulszähler",
+			"en" : "Operation Hours Pulses",
+			"fr" : "Compteur horaire"
+		}
+	},
+	{
+		"name"			: "workinghours",
```

Comment:
Jup!
Soll ich das in einem neuen PR machen? Ich würde dann auch den internen
Namen des Betriebsstundenzählers wieder auf Original zurücksetzen.



2016-04-06 16:26 GMT+02:00 andig <notifications@github.com>:

> In lib/Definition/EntityDefinition.json
> <https://github.com/volkszaehler/volkszaehler.org/pull/420#discussion_r58714215>
> :
>
> > @@ -240,6 +240,21 @@
> >  		"model"			: "Volkszaehler\\Model\\Channel",
> >  		"hasConsumption"	: true,
> >  		"translation"		: {
> > +			"de" : "Betriebsimpulszähler",
> > +			"en" : "Operation Hours Pulses",
> > +			"fr" : "Compteur horaire"
> > +		}
> > +	},
> > +	{
> > +		"name"			: "workinghours",
>
> Nennen wir diesen lieber workinghourstotal oder sowas?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/volkszaehler/volkszaehler.org/pull/420/files/50fea64589a2965a783fef203b3f02ec33a3f97e#r58714215>
>